plugins/Live/stylesheets/live.less

```diff
@@ -442,11 +442,17 @@ a.visitor-log-visitor-profile-link {
     position: absolute;
     right: 15px;
     top: 15px;
+    padding: 0 0 0.75em 0.75em;;
     font-style: italic;
     font-size: 13px;
     background-color: inherit !important;
     text-decoration: none;
 
+    @media (max-width: 600px) {
+      position: static;
+      display: block;
+    }
```

Comment:
ℹ️ Avoid stacking the link on top of the content.

| *before* | *after* |
|----|----|
| <img width="352" height="200" alt="Image" src="https://github.com/user-attachments/assets/bad753f3-354f-4737-839f-a1e373faaee8" /> | <img width="352" height="200" alt="Image" src="https://github.com/user-attachments/assets/c69faf87-319e-4c7a-bf89-b494cdbccc0f" /> |